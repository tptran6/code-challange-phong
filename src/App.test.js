import React from 'react'
import {render, cleanup} from '@testing-library/react'
import App from './App'

// Testing CI implementation
afterEach(cleanup)

test('snapshot', () => {
    const { asFragment } = render(<App />)

    expect(asFragment(<App />)).toMatchSnapshot()
});
