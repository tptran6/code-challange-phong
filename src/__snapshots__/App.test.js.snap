// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <h2>
    Random Post
  </h2>
  <div>
    <p>
      Loading...
    </p>
  </div>
</DocumentFragment>
`;
